### PR TITLE
Add fairseq S2T/S2S pipelines

### PR DESCRIPTION
**Pull request by kahne**

### CODE REVIEW

File: api-inference-community/docker_images/fairseq/app/pipelines/audio_to_audio.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        model, cfg, task = load_model_ensemble_and_task_from_hf_hub(
+            model_id,
+            arg_overrides={"config_yaml": "config.yaml"},
+            cache_dir=os.getenv("HUGGINGFACE_HUB_CACHE"),
+        )
+        self.model = model[0].cpu()
+        self.model.eval()
+        cfg["task"].cpu = True
+        self.task = task
+        self.generator = task.build_generator(self.model, cfg)
```
Comment (Narsil) [inline api-inference-community/docker_images/fairseq/app/pipelines/audio_to_audio.py:21]: This lines seemed to trigger an error in previous PR, 

`elf.generator = task.build_generator(model, cfg)` fixes it but maybe changing self.model into a list of models could work too.

**Review by Narsil — COMMENTED** [review]
This looks OK but:

- There's 1 potential bug (other comment)
- The model  `"facebook/xm_transformer_600m-es_en-multi_domain"` is private and not testable unfortunately.
- This PR probably needs to be rebase to benefits other fixes from https://github.com/huggingface/huggingface_hub/pull/569

**Comment by osanseviero** [comment]
On top of that please don't forget to run `make style` and `make quality` from `api-inference-community` :smile: Thanks!

File: api-inference-community/docker_images/fairseq/app/pipelines/audio_to_audio.py (hunk 1 above)
Comment (kahne) [reply api-inference-community/docker_images/fairseq/app/pipelines/audio_to_audio.py]: Thanks for catching this! I updated accordingly and also pulled the latest main branch.

**Comment by kahne** [comment]
> On top of that please don't forget to run `make style` and `make quality` from `api-inference-community` 😄 Thanks!

Thanks for the reminder! I updated the code with `black` and `isort` but am still getting test errors (it seems the CI test has a different result on `make quality` compared to local run of `make quality`). Do you have any idea?

**Pull request merged**